c++/include/orc/Reader.hh

```diff
@@ -149,6 +149,26 @@ namespace orc {
      */
     RowReaderOptions& includeTypes(const std::list<uint64_t>& types);
 
+    /**
+     * A map of <typeId, ArrayReadIntent> used as a parameter in
+     * RowReaderOptions::includeTypesWithIntents.
```

Comment:
Actually, this PR also uses it at `getReadIntents` too. We had better describe the definition a little more instead of listing the references.

Reply:
Deleted in latest commit.